src/lightning/pytorch/core/module.py

```diff
@@ -1640,86 +1640,6 @@ def _register_sharded_tensor_state_dict_hooks_if_available(self) -> None:
                 weakref.proxy(self), pre_load_state_dict_hook, True  # type: ignore[arg-type]
             )
 
-    @classmethod
-    def from_compiled(cls, model: "torch._dynamo.OptimizedModule") -> "pl.LightningModule":
```

Comment:
Weren't these meant to be actual methods on the LM? Maybe keep but call the function below?

Reply:
cc @lantiga: Could you let us know if you wrote these `classmethod`s for a specific benefit? The implementation does not need it. The motivation for moving it is just to improve the readability of `module.py`

Reply:
No specific benefit, just grouping. It’s 100% fine to move it.